examples/recipe_creator/backend/app.py

```
@@ -0,0 +1,118 @@
+        
+        def generate():
+            for chunk in stream_client.run(
+                ingredients=ingredients,
+                dietary_restrictions=dietary_restrictions,
+                time_limit=time_limit
+            ):
+                yield f"data: {json.dumps(chunk)}\n\n"
+        
+        return Response(generate(), mimetype='text/event-stream')
+    
+    except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)